plugins/modules/cmci_delete.py

```diff
@@ -44,14 +44,13 @@
 - name: delete a bundle definition in a CICS region
   cmci_delete:
     cmci_host: 'winmvs2c.hursley.ibm.com'
-    cmci_port: '10080'
+    cmci_port: 10080
     context: 'iyk3z0r9'
-    option: 'delete'
     type: CICSDefinitionBundle
     resource:
       filter:
         name: 'PONGALT'
-      parameters:
+      get_parameters:
```

Comment:
Is this one right or is it meant to be delete_parameters? Can't remember if we have a specific delete_parameters, and you've got rid of the delete option above.

Reply:
This one's right, parameters under `resources` are `get_parameters`, top level ones are `delete_parameters`.  It's not ideal, but that's really caused by them having to be supplied differently!

Reply:
ah got it, all good then 😄